onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -359,6 +360,8 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformers(
       }
 #endif
 
+      transformers.emplace_back(std::make_unique<MatMulNBitsFusion>(cpu_ep));
```

Comment:
>transformers.emplace_back(std::make_unique<MatMulNBitsFusion>(cpu_ep)); [](http://example.com/codeflow?start=6&length=71)

as neural_speed doesn't support bias, could you please add a guard for it?

Reply:
sure, added.